src/python-common/ceph/deployment/service_spec.py

```diff
@@ -849,6 +859,19 @@ def validate(self):
         if not self.keepalived_password:
             raise ServiceSpecValidationError(
                 'Cannot add HA_RGW: No Keepalived Password specified')
+        if self.ha_proxy_frontend_ssl_certificate:
+            if not self.ha_proxy_frontend_ssl_port:
+                raise ServiceSpecValidationError(
+                    'Cannot add HA_RGW: Specified Ha Proxy Frontend SSL Certificate but no SSL Port')
+            if not self.ha_proxy_ssl_dh_param:
+                raise ServiceSpecValidationError(
+                    'Cannot add HA_RGW: Specified Ha Proxy Frontend SSL Certificate but not SSL dh param')
+            if not self.ha_proxy_ssl_ciphers:
```

Comment:
Add checks in order to verify that values provided are "inside" the ones we support